Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [InnerTune/App](https://hosted.weblate.org/projects/innertune/android/).


It also includes following components:

* [InnerTune/Fastlane (app store details)](https://hosted.weblate.org/projects/innertune/fastlane-app-store-details/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/innertune/android/horizontal-auto.svg)

lucasmz-dev commented:
Don't merge this yet

lucasmz-dev commented:
Hmm it seems we're getting warnings about missing plurals

Seems related to https://github.com/WeblateOrg/weblate/issues/7520

lucasmz-dev commented:
Well I solved that issue by not letting Weblate update unneeded files, and not having the plugin to remove translations to no longer existinting strings/strings that don't exist in the original file; I think if I had just built it it would've worked just fine, but the linter complains for these languages for some reason

lucasmz-dev commented:
*Can* be merged if you want at this point

lucasmz-dev commented:
(you can use github's squash and merge here if you prefer to keep this under one commit, it's a bit spammy right now)

lucasmz-dev commented:
it's just linting breaking, building works
not sure how to deal with this 😞

lucasmz-dev commented:
#1573 *

lucasmz-dev committed:
Deleted translation using Weblate (Norwegian Bokmål)

lucasmz-dev committed:
Deleted translation using Weblate (Spanish (American))

doctorreditshere committed:
Translated using Weblate (Indonesian)

Currently translated at 78.3% (195 of 249 strings)

Translation: InnerTune/App
Translate-URL: https://hosted.weblate.org/projects/innertune/android/id/
